File: Data Structures & Algorithms/number-of-one-bits/submission-0.cpp
class Solution {
public:
    int hammingWeight(uint32_t n) {
        int count = 0;

        while ( n > 0) {
            int map = n & 1;
            if (map == 1) {
                count++;
            }

            n = n >> 1;
        }

        return count;
    }
};

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        frequencyMap = {}
        for num in nums:
            if num not in frequencyMap:
                frequencyMap[num] = 1
                continue
            frequencyMap[num] += 1

        frequencyHeap = []
        heapq.heapify(frequencyHeap)

        for num, count in frequencyMap.items():
            result = (-count, num)
            print(result)
            heapq.heappush(frequencyHeap, result)

        toReturn = []
        for i in range(k):
            count, num = heapq.heappop(frequencyHeap)
            toReturn.append(num)

        return toReturn
        